packages/@aws-cdk/aws-apigateway/test/domains.test.ts

```diff
@@ -265,22 +257,17 @@ describe('domains', () => {
         'Ref': 'apiDeploymentStageprod896C8101',
       },
     });
-
-
   });
 
   test('domain name cannot contain uppercase letters', () => {
     // GIVEN
     const stack = new Stack();
     const certificate = new acm.Certificate(stack, 'cert', { domainName: 'someDomainWithUpercase.domain.com' });
 
-    // WHEN
+    // WHEN & THEN
     expect(() => {
       new apigw.DomainName(stack, 'someDomain', { domainName: 'someDomainWithUpercase.domain.com', certificate });
-    }).toThrow(/uppercase/);
-
-    // THEN
-
+    }).toThrow(/Domain name does not support uppercase letters./);
```

Comment:
Curious why change this. The test intentionally used `/uppercase/` regex  because it allows the error message to be adjusted without having to change the test.

Could you revert this?

Reply:
Yep.. I can definitely revert it.